orga/translations/en.json

```diff
@@ -55,19 +55,37 @@
     },
     "campaign-details": {
       "actions": {
-        "edit": "Edit",
         "archive": "Archive",
         "copy-link": {
           "copied": "Copied!",
           "copy": "Copy direct link"
-        }
+        },
+        "edit": "Edit"
       },
       "direct-link": "Direct link",
       "external-user-id-label": "External user ID label",
       "landing-page-text": "Text to display on the starting page",
       "personalised-test-title": "Title of the personalised test",
       "target-profile": "Target profile"
     },
+    "profiles-list": {
+      "table": {
+        "column": {
+          "first-name": "First name",
+          "last-name": "Last name",
+          "sending-date": "Sending date",
+          "pix-score": "Pix score",
+          "certifiable": "Eligible for certification",
+          "competences-certifiables": "Certifiable skills"
+        },
+        "empty": "Waiting for profiles",
+        "row-title": "Profile"
+      },
+      "status": {
+        "certifiable": "Eligible for certification",
```

Comment:
ce statut n'est pas utile, "Eligible for certification" est déjà dans "table.column.certifiable"